src/wallet/walletdb.cpp

```diff
@@ -624,6 +671,23 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
             }
             wss.fIsEncrypted = true;
         }
+        else if (strType == "csapzkey")
+        {
+            libzcash::SaplingPaymentAddress addr;
+            ssKey >> addr;
+            libzcash::SaplingFullViewingKey fvk;
+            ssValue >> fvk;
+            vector<unsigned char> vchCryptedSecret;
+            ssValue >> vchCryptedSecret;
+            wss.nCKeys++;
+
+            if (!pwallet->LoadCryptedSaplingZKey(fvk ,vchCryptedSecret, addr ))
```

Comment:
Likewise, we won't know `addr` here, so the last parameter needs to be optional. Also whitespace.